lib/git/rev_list.rb

```diff
@@ -0,0 +1,36 @@
+require 'git'
+
+module LibRevList
+  def rev_list(committish = nil, opts = {})
+    arr_opts = []
+
+    opts.each do |k, v|
+      # allow for passing, say, :max-count or :max_count
+      k = k.to_s
+      k.tr!('_', '-')
+      if v && v.to_s.downcase == 'true' # rubocop:disable Performance/Casecmp
```

Comment:
So, here's the deal with `Performance/Casecmp` : We have three options.

1. We do what I did above and disable the cop here
2. We use `casecmp` instead of downcase, _but_ that means the new line would look like:
`if v && v.to_s.casecmp('true') == 0`
which does not look better to me. There is a casecmp? function available that returns a bool and not an integer, but only in ruby > 2.4
3. We disable that cop because it's asking us to do something that makes things look strange.

Reply:
I'm fine with 3.